src/main/scala/nl/knaw/dans/easy/multideposit/package.scala

```diff
@@ -124,7 +124,6 @@ package object multideposit {
       * Writes a CharSequence to a file creating the file if it does not exist using the default encoding for the VM.
       *
       * @param data the content to write to the file
-      * @throws IOException in case of an I/O error
```

Comment:
i agree. it seems to be a scaladoc bug. i see two options: 
- add a -no-link-warnings to the scaladoc, 
- or remove the throws-clauses from the docs

@janvanmansum has opted for the second one. i've added throws annotations to keep the reference in the code